src/Model/GroundWaterTransport/gwt1cnc1.f90

```diff
@@ -115,24 +121,28 @@ subroutine cnc_allocate_arrays(this, nodelist, auxvar)
       this%ratecncin(i) = DZERO
       this%ratecncout(i) = DZERO
     end do
+    ! -- set constant head array input context pointer
+    call mem_setptr(this%tspvar, 'TSPVAR', this%input_mempath)
```

Comment:
Presumably, you cal this tspvar instead of concentration because you are planning to move this to the general TSP model?

Reply:
Yes, the thought behind `tspvar` (short for _transport variable_) is that eventually it will support either concentration or temperature.  We can rename now or later if you think that would be helpful.